pkg/api/v1/zz_generated.deepcopy.go

```diff
@@ -973,6 +973,7 @@ func (in *AwsKms) DeepCopyInto(out *AwsKms) {
 		*out = new(bool)
 		**out = **in
 	}
+	out.SecretRef = in.SecretRef
```

Comment:
Side comment here. I keep on seeing changes that I think come from the linter in some autogenerated files, I need to manually revert those. Anyone got the same or is just me?